AsyncDisplayKit/Layout/ASLayoutSpec.mm

```diff
@@ -156,14 +162,21 @@ - (void)setChildren:(NSArray *)children
 
 - (id<ASLayoutable>)child
 {
-  return _child;
+  return [_children firstObject];
```

Comment:
Ah, very cool!  

> On Jun 2, 2016, at 10:08 PM, ricky <notifications@github.com> wrote:
> 
> In AsyncDisplayKit/Layout/ASLayoutSpec.mm <https://github.com/facebook/AsyncDisplayKit/pull/1693#discussion_r65657217>:
> 
> > @@ -156,14 +162,21 @@ - (void)setChildren:(NSArray *)children
> >  
> >  - (id<ASLayoutable>)child
> >  {
> > -  return _child;
> > +  return [_children firstObject];
> firstObject won't crash. It will return nil.
> https://developer.apple.com/library/mac/documentation/Cocoa/Reference/Foundation/Classes/NSArray_Class/#//apple_ref/occ/instp/NSArray/firstObject <https://developer.apple.com/library/mac/documentation/Cocoa/Reference/Foundation/Classes/NSArray_Class/#//apple_ref/occ/instp/NSArray/firstObject>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/facebook/AsyncDisplayKit/pull/1693/files/501fed16aa2e798117d773774bdaaa4d0bac17ed#r65657217>, or mute the thread <https://github.com/notifications/unsubscribe/AAigA7xk9KGO20G_s2HtMLculY3dhgLHks5qH7bMgaJpZM4Ik-md>.
>